src/main/index.ts

```diff
@@ -38,13 +38,23 @@ const getUpgradeToVersion = async (foundVersions: FoundVersion[], dependency, op
 const resolveDependencyParam = (dependencyParam: string) => {
     if (dependencyParam.startsWith("@")) {
         const [_, name, version] = dependencyParam.split("@")
-        return { name: "@" + name, version }
+        return { name: "@" + name, version, isDevDependency: false }
     } else {
         const [name, version] = dependencyParam.split("@")
-        return { name, version, foundVersions: [], upgradeToVersion: "" }
+        return { name, version, foundVersions: [], upgradeToVersion: "", isDevDependency: false }
     }
 }
 
+const isDevDependency = (foundVersions: FoundVersion[]): boolean => {
+    let isDevDependency: boolean
+
+    foundVersions.forEach(({ packages }) => {
+        isDevDependency = packages.some(({ dependencyAttribute }) => dependencyAttribute === "devDependencies")
+    })
+
+    return isDevDependency
+}
```

Comment:
**question:** Not sure this does what you intended? It will check all found versions but only use the result of the latest one to check if some is equals to `devDependencies`. 

Also I feel like this entire logic would need a definition. If multiple versions are found in different parents, should we lean towards one or the other? I would lean towards putting on `dependencies`. So if all found versions in all packages are in devDependencies then I think we can automatically dedupe to `devDependencies`, otherwise to dependencies.

Reply:
Yeah reading your comment and looking at it again I realize the logic is flawed.  
My original thought was the opposite of how you were thinking of it -- I was thinking about it along the lines that if "any" of the versions is a "devDependencies" then we want to make it a "devDependencies" when added to root but I realized there are flaws to this line of thinking.  Making it a "devDependencies" only if ALL versions in all packages are a "devDependency" is the better approach (which is what you suggested 👍🏽 ).  I've made the update inline with this approach!